generators/client-2/templates/src/main/webapp/app/shared/auth/_auth.service.ts

```diff
@@ -2,9 +2,11 @@ import { Injectable, Inject } from '@angular/core';
 import { StateService } from 'ui-router-ng2';
 import { LoginService } from "../login/login.service";
 import { Principal } from './principal.service';
-import { AuthServerProvider } from './auth-session.service';
-import { <%=jhiPrefixCapitalized%>TrackerService } from '../tracker/tracker.service';
-
+import { AuthServerProvider } from  <%_ if (authenticationType === 'jwt') { _%> './auth-jwt.service';
```

Comment:
For inline template `<%_` shouldn't be used. Thats for neline templates. Use `<%` for inline